### ISSUE TITLE
‘Recode’ is save in batch file?

### ISSUE THREAD

**Issue by MilagrosGMarin** [sdcTools/UserSupport#89]
*Please specify*  
**SDC tool used: tau-argus
**Version used: 4.1.7 Build 4  
**Operating system used: Windows

This is a sequence of actions that I do, and next I save in a batch file (Test01.arb)
1º Open microdata
2º Specify table
3º Do the rule
4º ‘Recode’ , for example: 1:1-3 (in file Recode1. grc) 
5º ‘Supress’ ‘Hypercube’
6º ‘Audit’
When I open this batch file (File --> Open batch process..) , the ‘recode’ has disappeared 
Is it like that?
Is it possible to keep the recoding?
Thank,
Milagros..

**Comment by AncoHundepool** [comment]
Dear Milagros,

The issue is that recode is not a command in the batch version of Tau-Argus.
That is why the recode is not generated when you write a batch-file.
It is on our wish/todo list, but it has not yet been implemented.

And also the audit is not part of the batch version. But that is not (yet) on our wish'/todo list.

**Comment by ppdewolf** [comment]
This now more a feature request 😃

**Comment by g3o2** [comment]
Just a note here to support this feature request.

It would probably require three arguments:

```generic
<READGLOBALRECODE> (TableNumber, RecodeableVariable, FilePath)
```

**Comment by ppdewolf** [comment]
This is already included in version 4.2.4. See the [release notes 4.2.4](https://github.com/sdcTools/tauargus/releases/download/v4.2.4.2/ReleaseNoteTAU4.2.4.pdf) how to use it.
